content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -138,7 +138,8 @@ The sum of their values will account for the total amount of reserved memory.
 A new `--reserved-memory` flag was added to Memory Manager to allow for this total reserved memory
 to be split (by a node administrator) and accordingly reserved across many NUMA nodes.
 
-The flag specifies a comma-separated list of memory reservations per NUMA node.
+The flag specifies a comma-separated list of memory reservations of different memory types per NUMA node.
```

Comment:
should `comma` be semi-colon here?

Reply:
No, comma is used as separator for different memory type reservations in same numa node.

Reply:
ah!